test/prime generator/prime.chpl

```diff
@@ -0,0 +1,46 @@
+/* Primality Test */
```

Comment:
I believe the underscore is important to prevent at least one of our test systems from getting confused about the pathname (cygwin, iirc), but I could be misremembering.  Please use either an underscore or camelCase naming for the directory